magento/magento2
# [MFTF] Refactoring of AdminMassUpdateProductAttributesMissingRequiredFieldTest

### Description (*)
The test is refactored according to the best practices followed by MFTF.


### Manual testing scenarios (*)
1. Create 2 simple products
2. Go to Admin->Catalog->Products
3. Request mass update for product name
4. Do not fill the "Product Name"
5. Check if the "Product Name" field is required




### Resolved issues:
1. [x] resolves magento/magento2#31160: [MFTF] Refactoring of AdminMassUpdateProductAttributesMissingRequiredFieldTest
